volontulo/templates/homepage.html

```diff
@@ -49,14 +34,44 @@
                             </div>
                         </div>
                         <h3><a class="text-warning" href="{% url 'offers_view' o.title|slugify o.id %}">{{ o.title }}</a></h3>
-                        <div class="text-right">Organizator:<a class="text-warning" href="{% url 'organization_view' slug=o.organization.name|slugify id_=o.organization.id %}">{{ o.organization.name }}</a></div>
+                        <div class="text-right">Organizator: <a class="text-warning" href="{% url 'organization_view' slug=o.organization.name|slugify id_=o.organization.id %}">{{ o.organization.name }}</a></div>
                     </div>
                 </div>
             </div>
         {% endfor %}
         </div>
     {% else %}
-        <p>Brak ofert spełniających podane kryteria</p>
+        <p class="lead">Zgłoś swoją ofertę aby znaleźć wolontariuszy do pomocy w akcji!</p>
+        {% if user.is_anonymous and not user.is_authenticated %}
+        <div class="media">
+            <div class="media-left media-middle">
```

Comment:
Every div has media-left media-middle. What about moving this to parent div so:
```css
.media.media-left.media-middle>div {}
```
Another inconsistency is that here is used: left and middle. I suppose that it comes from LEFT center right and top MIDDLE bottom positioning but it could be sometimes a little confusing.